README.md

```diff
@@ -196,6 +205,10 @@ make bats
 make: 'bats' is up to date.
 ```
 
+`publish` allows the publication of all images targeted by 'linux' to a registry.
+
+`ci-init` is dedicated to Jenkins infrastructure for initialising docker and isn't required in other contexts.
```

Comment:
```suggestion
`buildx-init` is dedicated to Jenkins infrastructure for initializing docker and isn't required in other contexts.
```

Reply:
or

```suggestion
`ci-init` is dedicated to Jenkins infrastructure for initializing docker and isn't required in other contexts.
```

Reply:
Cf comment on #421,

> I chose the English spelling: https://sapling.ai/usage/initialize-vs-initialise, is that OK?

Reply:
No problem for me, *but* I was reminded by @MarkEWaite that we prefer US English by convention in https://github.com/jenkins-infra/jenkins.io/pull/7358#discussion_r1635282081.
I should have added this link earlier, my bad

Reply:
Good to know, thanks for the link!